applications/adv_networking_bench/cpp/main.cpp

```diff
@@ -33,66 +33,66 @@ class App : public holoscan::Application {
     using namespace holoscan;
```

Comment:
Might as well add the networking namespace here like you did for the other files

Reply:
kept it like this and removed `holoscan::` from the current calls